index.html

```diff
@@ -17,14 +17,17 @@
           </div>
 
           <nav class="desktop-nav">
+            <div class="x-nav">
+              <img src="images/closeMark.png" alt="close mark" />
+            </div>
             <ul>
-              <li><a href="#">Portfolio</a></li>
-              <li><a href="#">About</a></li>
-              <li><a href="#">Contact</a></li>
+              <li><a id="ancreX" href="#ancre1">Portfolio</a></li>
+              <li><a href="#ancre2">About</a></li>
+              <li><a href="#ancre3">Contact</a></li>
```

Comment:
- [ ] Great effort on making these changes requested by the previous reviewer :+1: unfortunately it doesn't still work as expected. When clicking on any mobile menu options, the correct part of the page is not still displayed. You need to include all these ids in the various sections of the page. For example, you have the link  `href="#ancre2"`, in order to navigate to the about section of the page when clicks on the About link, you will need to add this id(ancre2) in the about section of your page. Get to line 184 and do this  `<section id="ancre2" class="About-section">` . Please do the same for the rest.